crates/uv/tests/it/help.rs

```diff
@@ -450,6 +452,10 @@ fn help_subcommand() {
 
 #[test]
 fn help_subsubcommand() {
+    if env::var("UV_PYTHON_INSTALL_DIR").is_ok() {
+        // skip this test
+        return;
+    }
```

Comment:
Note, you can use instead

`uv_snapshot!(context.filters(), context.help().env_remove(EnvVars::UV_PYTHON_INSTALL_DIR).arg("python").arg("install"), @r###"`